src/main/java/blackjack/view/InputView.java

```diff
@@ -0,0 +1,47 @@
+package blackjack.view;
+
+import blackjack.domain.user.Dealer;
+import blackjack.domain.user.Name;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Scanner;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class InputView {
+    private static final Scanner SCANNER = new Scanner(System.in);
+    private static final String ASK_PLAYERS_NAME_MSG = "게임에 참여할 사람의 이름을 입력하세요. (쉼표 기준으로 분리)";
+    private static final String DELIMITER = ",";
+    private static final String MORE_DRAW_MSG_FORMAT = "%s는 한장의 카드를 더 받겠습니까?(예는 y, 아니오는 n)%n";
+    private static final String YES = "y";
+    private static final String NO = "n";
+    private static final String ANSWER_MUST_BE_YES_OR_NO_ERROR_MSG_FORMAT = "%s 또는 %s 로 입력해주세요.";
+
+    public static List<String> askPlayersName() {
+        System.out.println(ASK_PLAYERS_NAME_MSG);
+        return trimNames(splitNames(SCANNER.nextLine()));
+    }
+
+    private static List<String> splitNames(String names) {
+        return Arrays.asList(names.split(DELIMITER));
+    }
+
+    private static List<String> trimNames(List<String> splitNames) {
+        return splitNames.stream().map(String::trim).collect(Collectors.toList());
```

Comment:
```suggestion
        return splitNames.stream()
                .map(String::trim)
                .collect(Collectors.toList());
```

stream 로직은 한줄에 한 행위씩 작성해주세요 ㅎㅎ 전체적으로 한번 체크해주세요! :)

Reply:
이정도 길이는 읽히지 않나 하고 작성을 했었는데, 한줄에 한 행위씩 👍  명심하겠습니다